recipes/graphium/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "graphium" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/graphium-{{ version }}.tar.gz
```

Comment:
Personally I prefer the GitHub source as it is closer to the source, increasing the changes not running out-of-date. But if you are the maintainer of the library it could be fine if you know for sure that the upload to pypi is automatic

Reply:
Thank you for taking the time to do the review. This is the first time I am hearing about gh being prefered over pypi. Usually pypi is always prefered over gh for source archives. Is it a new recommendation? If not I would prefer sticking with pypi

Reply:
No. It's my personal preference, it's perfectly fine to leave it as it is